src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/loadbalancer/ApacheEc2LoadBalancerFormula.kt

```diff
@@ -13,7 +14,9 @@ import org.apache.logging.log4j.Logger
 import java.net.URI
 import java.time.Duration
 
-internal class ApacheEc2LoadBalancerFormula : LoadBalancerFormula {
+class ApacheEc2LoadBalancerFormula : LoadBalancerFormula {
```

Comment:
Theoretically we are adding new API here so it looks like this should belong to `### Added` section and a minor bump.

Reply:
Done.